src/components/GlistenClient.vue

```diff
@@ -139,6 +138,18 @@ export default class GlistenClient extends Vue {
   @Prop({ required: true })
   public applicationId!: string;
 
+  // Greetings displayed at the top of the component
+  @Prop({ required: false, default: 'Thanks for letting us know how do you like this and sending your feedback to further improve.' })
```

Comment:
let's put this as a default message: Thank you so much for taking the time to share your feedback with us! We appreciate hearing your thoughts on how we're doing, and we're excited to use your feedback to become even better.